app/src/pages/explorer/Curate.vue

```diff
@@ -39,7 +39,16 @@
                                 </div>
                             </router-link>
                         </div>
-                        <!-- <div class="md-layout-item">&nbsp;</div> -->
+                        <div class="md-layout-item md-layout-item_card">
+                            <router-link to="" v-slot="{navigate, href}" custom>
+                                <div class="teams_container explorer_page-nav-card md-layout-item_card" :href="href" @click="navigate">
+                                    <md-icon class="explorer_page-nav-card_icon">article</md-icon>
+                                    <span class="explorer_page-nav-card_text">Submit FEA Templates</span>
```

Comment:
I may have misunderstood, but I thought Bingyin said that FEA uses the same script as XML so aside from needing to add a link to the blank template and to compression in supplementary materials we can use the same spreadsheet upload page